ui/src/api-logic/mappers/imports.ts

```diff
@@ -0,0 +1,27 @@
+import { RepoImportData } from 'src/@types'
+import { SYNC_REPO_METHOD } from 'src/utils/constants'
+import { GetRepoImportsQuery } from '../graphql/generated/schema'
+
+/**
+ * Method which iterate each repo import and map it to RepoImportData to be shown in table
+ * @param data Repo import list that comes from data base in GetRepoImportsQuery format
+ * @returns Repo import list from data base mapped to RepoImportData list
+ */
+const mapToImpotsData = (data: GetRepoImportsQuery | undefined): Array<RepoImportData> => {
```

Comment:
```suggestion
const mapToImportsData = (data: GetRepoImportsQuery | undefined): Array<RepoImportData> => {
```

Seems like a missing `r` typo?

Reply:
Done [414f4b3](https://github.com/mergestat/fuse/pull/200/commits/414f4b30543eda7033cd30d4eded4b4df6c21afc)